repose-aggregator/components/filters/ip-user/src/main/scala/org/openrepose/filters/ipuser/IpUserFilter.scala

```diff
@@ -28,23 +28,22 @@ import javax.servlet.http.{HttpServletRequest, HttpServletResponse}
 import com.typesafe.scalalogging.slf4j.LazyLogging
 import edazdarevic.commons.net.CIDRUtils
 import org.openrepose.commons.config.manager.UpdateListener
-import org.openrepose.commons.utils.servlet.http.MutableHttpServletRequest
+import org.openrepose.commons.utils.servlet.http.HttpServletRequestWrapper
 import org.openrepose.core.filter.FilterConfigHelper
 import org.openrepose.core.services.config.ConfigurationService
 import org.openrepose.filters.ipuser.config.IpUserConfig
 
 @Named
 class IpUserFilter @Inject()(configurationService: ConfigurationService) extends Filter
-with LazyLogging
-with UpdateListener[IpUserConfig] {
+  with LazyLogging
+  with UpdateListener[IpUserConfig] {
 
   private final val DEFAULT_CONFIG = "ip-user.cfg.xml"
-  private var initialized = false
-  private var configName: String = _
-
-  case class LabeledCIDR(label: String, cidr: CIDRUtils)
 
   private val cidrList: AtomicReference[List[LabeledCIDR]] = new AtomicReference[List[LabeledCIDR]]()
```

Comment:
Unrelated to your changes...  I've been wondering if we should be doing this in the other filters since we're updating a shared object (cidrList in this case) without worrying about the object being half built when another thread tries to use it.